client/connect.go

```diff
@@ -1,38 +1,47 @@
 package client
 
 import (
+	"github.com/MagalixTechnologies/channel"
 	"os"
-	"strings"
+	. "strings"
```

Comment:
Keep the imported package name for readability

Reply:
Done